### ISSUE TITLE
BadRequest on MultiSearch leads to NullPointerException

### ISSUE THREAD

**Issue by Dbzman** [sksamuel/elastic4s#1559]
Hey there,

we accidentally stumbled accross a BadRequest. Sadly elastic4s did not show this correctly. A NullPointerException is thrown there when using MultiSearch. ([This line breaks](https://github.com/sksamuel/elastic4s/blob/022b56d01d37945ff6b697b33ec1567732aae33b/elastic4s-http/src/main/scala/com/sksamuel/elastic4s/http/search/SearchHandlers.scala#L25))
It tries to get the responses object but it does not exist in case of a 400.
Example Response:
```json
{
  "error": {
    "root_cause": [
      {
        "type": "parsing_exception",
        "reason": "random_score query does not support [field]",
        "line": 1,
        "col": 163
      }
    ],
    "type": "parsing_exception",
    "reason": "random_score query does not support [field]",
    "line": 1,
    "col": 163
  },
  "status": 400
}
```

Cheers
Timo

**Comment by vn971** [comment]
Confirming this. There is a NullPointerException for response entities that have no "responses" json field.

**Comment by sksamuel** [comment]
Thanks, now fixed in master for 7.0 release.
